### PR TITLE
fix: inline classes table

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

Modified data from _data/*.json to simplify displaying data on table

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: docs/.vuepress/baseComponents/ComponentClassTable.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@
           .{{ i.class }}
         </th>
         <td class="d-ff-mono d-fc-orange d-fs12">
-          <span class="code-example--inline">{{ i.applies !== 'N/A' ? `.${i.applies}` : i.applies }}</span>
+          <span class="code-example--inline" v-html="i.applies"/>
```
Comment (braddialpad) [inline docs/.vuepress/baseComponents/ComponentClassTable.vue]: is there a reason for this to be `v-html`? can't it just be `{{i.applies}}`?

Comment (braddialpad) [reply docs/.vuepress/baseComponents/ComponentClassTable.vue]: Couple linter errors here too. hmm not sure how they got through with lint-staged running.

Comment (juliodialpad) [reply docs/.vuepress/baseComponents/ComponentClassTable.vue]: > is there a reason for this to be `v-html`? can't it just be `{{i.applies}}`?

We have [some html](https://github.com/dialpad/dialtone/blob/e0bc251c73251589657ca0de5e3e4ad6809731ad/docs/_data/modal.json#L10) in there but I think I can remove it, it's not making anything IMO

Comment (juliodialpad) [reply docs/.vuepress/baseComponents/ComponentClassTable.vue]: > Couple linter errors here too. hmm not sure how they got through with lint-staged running.

Weird, let me check it

Comment (braddialpad) [reply docs/.vuepress/baseComponents/ComponentClassTable.vue]: > We have [some html](https://github.com/dialpad/dialtone/blob/e0bc251c73251589657ca0de5e3e4ad6809731ad/docs/_data/modal.json#L10) in there but I think I can remove it, it's not making anything IMO

ah yeah, guess we can remove

File: docs/.vuepress/baseComponents/ComponentClassTable.vue [hunk 2]
```diff
@@ -31,7 +31,10 @@
           .{{ i.class }}
         </th>
         <td class="d-ff-mono d-fc-orange d-fs12">
-          <span class="code-example--inline">{{ i.applies !== 'N/A' ? `.${i.applies}` : i.applies }}</span>
+          <span
+            class="code-example--inline"
+            v-text="i.applies"
+          />
```
Comment (braddialpad) [inline docs/.vuepress/baseComponents/ComponentClassTable.vue]: Does this also need to be updated in ComponentAccessibleTable?

Comment (juliodialpad) [reply docs/.vuepress/baseComponents/ComponentClassTable.vue]: Right, I'll update both to use the same syntax

**Review by braddialpad — APPROVED** [review]

**Pull request merged**